docs/source/manual/configuration.rst

```diff
@@ -69,7 +69,7 @@ GZip
 .. code-block:: yaml
 
     server:
-      gzip: 
```

Comment:
Please, don't reformat lines that are not relevant to the change. 
It makes the change less focused and more hard to review.